scalableminds/webknossos
# Sorting mappings in UI

This PR sorts the mappings listed in the dropdown of the segmentation info tab.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open a tracing and open the segmentation tab
- enable mappings by activating the check button
- the listed mappings in the dropdown below the button should now be sorted.

### Issues:
- fixes #3835

FYI:
- I needed to create a copy of the mappings of the segmenatation layer because sort sorts inplace and the mappings array is set to rread only.
------
